### PR TITLE
Add 1.3 release notes

### PR DESCRIPTION
**Pull request by eugenesk24**
### Description
Add 1.3 release notes
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-sql.release-notes-1.3.0.0.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.3.0
+
+### Features
+* Add parse command to PPL ([#411](https://github.com/opensearch-project/sql/pull/411))
+* PPL integration with AD and ml-commons ([#468](https://github.com/opensearch-project/sql/pull/468))
+
+### Enhancements
+* Support ISO 8601 Format in Date Format. ([#460](https://github.com/opensearch-project/sql/pull/460))
+* Treating ExpressionEvaluationException as client Error. ([#459](https://github.com/opensearch-project/sql/pull/459))
```
Comment (vamsi-amazon) [inline release-notes/opensearch-sql.release-notes-1.3.0.0.md:11]: Can you move this to bug fix.

**Review by anirudha — APPROVED** [review]
pls make vamsi's changes

**Review by penghuo — APPROVED** [review]
Thanks!

**Comment by joshuali925** [comment]
i think backport workflow doesn't work on merged commits, only squashed
```
/usr/bin/git cherry-pick -x fff24aaca82fba99b54fb14f9bc2a6[23](https://github.com/opensearch-project/sql/runs/5578233452?check_suite_focus=true#step:3:23)d1a3e566
  error: commit fff24aaca82fba99b54fb14f9bc2a623d1a3e566 is a merge but no -m option was given.
  fatal: cherry-pick failed

**Pull request merged**